dashboard/app/models/script.rb

```diff
@@ -342,6 +342,31 @@ def hoc_finish_url
     end
   end
 
+  def summarize
+    summary = {
+      id: id,
+      name: name,
+      stages: []
+    }
+    if trophies
+      summary[:trophies] = Concept.cached.map do |concept|
+        {
+          id: concept.name,
+          name: I18n.t('data.concept.description', concept.name),
```

Comment:
I think we broke data_t....

This isn't the same as what data_t does, and results in an array where we'd expect a string!